### ISSUE TITLE
podman logs and docker logs treat container stderr differently

### ISSUE THREAD

**Issue by tmds** [containers/podman#9102]
`docker logs` will print `stderr` from a container to `stderr`, and `podman logs` prints it to `stdout`.

Small reproducer:
`Dockerfile`:
```Dockerfile
FROM busybox

COPY script.sh /script.sh

CMD /script.sh
```
`script.sh`:
```sh
#!/bin/sh

echo "hello stderr" >/dev/stderr
```

Run:
```sh
$ [podman/docker] build .
$ [podman/docker] run <image>

# get logs, ignore stdout
$ [podman/docker] logs <container> >/dev/null
```
`docker` outputs `hello stderr`, and `podman` does not.

**Comment by Luap99** [comment]
This should be fixed in 3.0.0

**Comment by tmds** [comment]
@Luap99 great, thank you!
